Sources/Logging.php

```diff
@@ -442,7 +442,7 @@ function logActions($logs)
 
 	foreach ($logs as $log)
 	{
-		if ((empty($modSettings[$log['log_type'] . 'log_enabled']) || !isset($log_types[$log['log_type']])) && !in_array($log['action'], $always_log))
+		if (!in_array($log['action'], $always_log) && (empty($modSettings[$log['log_type'] . 'log_enabled']) || !isset($log_types[$log['log_type']])))
```

Comment:
`in_array($log['action'], $always_log)` instead of `!in_array($log['action'], $always_log)` to get moderation log work properly.

Reply:
Yes, with this correction Moderation Log now is working properly.